pkg/controller/bootstrap/bootstrapsigner_test.go

```
@@ -60,13 +64,13 @@ func newConfigMap(tokenID, signature string) *v1.ConfigMap {
 }
 
 func TestNoConfigMap(t *testing.T) {
-	signer, cl := newBootstrapSigner()
+	signer, cl, _ := newBootstrapSigner()
```

don't ignore the error.

---

ok